package.json

```diff
@@ -7,6 +7,7 @@
     "js-cookie": "^3.0.0",
     "leaflet": "^1.6.0",
     "leaflet.locatecontrol": "^0.75.0",
+    "osm-community-index": "^5.1.3",
```

Comment:
Also, current version is 5.1.6.

Reply:
The ^ at the start means compatible with (expecting semver), so running `npm i` would install 5.1.6
(for more info see: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies)